src/datetimepicker.windows.js

```diff
@@ -0,0 +1,64 @@
+/**
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License.
+ * @format
+ * @flow strict-local
+ */
+'use strict';
+
+import {requireNativeComponent, StyleSheet} from 'react-native';
+import type {WindowsNativeProps, WindowsDatePickerChangeEvent} from './types';
+import * as React from 'react';
+
+const styles = StyleSheet.create({
+  rnDatePicker: {
+    height: 32,
+    width: 150,
+  },
+});
+
+const RNDateTimePickerWindows = requireNativeComponent('RNDateTimePickerWindows');
+
+export default class DateTimePickerWindows extends React.Component<WindowsNativeProps> {
+  static defaultProps: WindowsNativeProps = {
+    dateFormat: 'day month year',
+    dayOfWeekFormat: '{dayofweek.abbreviated(3)}'
+  };
+
+  constructor(props: WindowsNativeProps) {
+    super(props);
+  }
+  render() {
+    const props = {
+      dayOfWeekFormat: this.props.dayOfWeekFormat,
+      dateFormat: this.props.dateFormat,
+      firstDayOfWeek: this.props.firstDayOfWeek,
+      maxDate: this.props.maximumDate ? this.props.maximumDate.getTime() : undefined, // time in milliseconds
+      minDate: this.props.minimumDate ? this.props.minimumDate.getTime() : undefined, // time in milliseconds
+      onChange: this.props.onChange,
+      placeholderText: this.props.placeholderText,
+      selectedDate: this.props.value
+        ? this.props.value.getTime()
+        : undefined, // time in milliseconds
+      style: [styles.rnDatePicker, this.props.style],
+    };
+
+    // The Date object returns timezone in minutes. Convert that to seconds
+    // and multiply by -1 so that the offset can be added to GMT time to get
```

Comment:
actually you don't want GMT you want UTC+0 (since GMT can go into summer time too)

Reply:
Copied from current implementation. I think this comment uses GMT liberally and just means timezone-aware time - I don't see any references to UTC/GMT here. What do you think?

Reply:
UTC is the correct terminology :) let's update it wherever it applies. Having worked on timezones, it's important to be accurate :)